package.json

```diff
@@ -33,10 +33,10 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-stage-0": "^6.3.13",
     "eslint": "^1.10.3",
-    "webpack": "^3.0.0",
-    "webpack-dev-server": "^2.5.1",
     "glob": "^7.1.1",
-    "url": "^0.11.0"
+    "url": "^0.11.0",
+    "webpack": "^3.0.0",
+    "webpack-dev-server": "^2.5.1"
```

Comment:
NPM jambles dependencies by putting them in alphabetical order- I don't mind the change for now, put let's be mindful of omitting changes like these in the future